dll/win32/shell32/shellrecyclebin/recyclebin_v5_enumerator.c

```diff
@@ -210,6 +210,41 @@ RecycleBin5File_RecycleBinFile_GetFileName(
     return S_OK;
 }
 
+static HRESULT STDMETHODCALLTYPE
+RecycleBin5File_RecycleBinFile_GetTypeName(
+    IN IRecycleBinFile *This,
+    IN SIZE_T BufferSize,
+    IN OUT LPWSTR Buffer,
```

Comment:
Unnatural ordering. Standard APIs usually do the order: Buffer, then existing Buffer size.

Reply:
Is it expected that this function always store a NULL-terminated string inside the buffer?

Reply:
Then we need to refactor all the original functions, all use the same format, and the null terminated is the same for all functions, i just used the above function to create my own function

Reply:
OK then don't change anything ^^ (regarding the ordering of parameters)

Reply:
so, not, it's not expected